src/products/components/ProductVariantChannels/ChannelsAvailabilityCard/AvailabilityCard.tsx

```diff
@@ -1,31 +1,45 @@
+import {
+  ChannelPriceAndPreorderData,
+  IChannelPriceAndPreorderArgs,
+} from "@dashboard/channels/utils";
 import { Divider } from "@dashboard/components/Divider";
+import { FormsetData } from "@dashboard/hooks/useFormset";
 import React from "react";
 
-import { Channel, ProductChannelListing } from "./../types";
+import { ProductChannelListing } from "./../types";
 import { ChannelsListItem } from "./ChannelsListItem";
 import CardContainer from "./VariantDetailsChannelsAvailabilityCardContainer";
 
 interface AvailabilityCardProps {
-  items: Channel[];
+  listings: FormsetData<
```

Comment:
suggestion: wouldn't be better to name it like this: `allListings` or `allAvailableListings`